src/Codeception/Module/Symfony2.php

```diff
@@ -358,8 +358,17 @@ public function seeCurrentRouteIs($routeName, array $params = [])
         if (!$router->getRouteCollection()->get($routeName)) {
             $this->fail(sprintf('Route with name "%s" does not exists.', $routeName));
         }
-        $url = $router->generate($routeName, $params);
-        $this->seeCurrentUrlEquals($url);
+
+        try {
+            $match = $router->match(explode('?', $this->grabFromCurrentUrl())[0]);
+        } catch (\Symfony\Component\Routing\Exception\ResourceNotFoundException $e) {
+            $this->fail(sprintf('The "%s" url does not match with any route', $routeName));
```

Comment:
Both these methods end up outputting an incorrect message because the URL should be placed in the string instead of the route name. Or the message should be reworded.